app/views/planning_application_mailer/invalidation_notice_mail.text.erb

```diff
@@ -15,25 +12,18 @@ To review the reasons for invalidation and make the changes required, please fol
 
 <%= @planning_application.secure_change_url(@planning_application.id, @planning_application.change_access_id) %>
 
-If your amendments contain further errors, your application will be returned to you again.
+If your amendments contain further errors, we may request further changes to your application.
 
 If we don’t hear from you:
 
-We will take no further action on your application until you make the necessary the change(s) to your application.
+We will take no further action on your application until you make the necessary change(s).
 
-If these change(s) are not received by <%= @planning_application.invalidation_response_due.to_formatted_s(:day_month_year) %>, your application will be returned to you and your payment refunded.
+If these change(s) are not received by <%= @planning_application.invalidation_response_due.strftime('%d %B %Y') %>, your application will be returned to you and your payment refunded.
```

Comment:
this is `day_month_year`?

Reply:
yes, like so: 
![image](https://user-images.githubusercontent.com/9452321/132380777-fdb05ddf-3066-4f72-9abd-36d2bf714687.png)

Reply:
that day_month_year method wasn't displaying the date correctly so I reached for strftime 🤷

Reply:
what does "not displaying the date correctly" mean? They're exactly the same format except ours doesn't have the zero padding.

Reply:
![image](https://user-images.githubusercontent.com/9452321/132382421-4b3f79d3-2e43-4814-9a6c-6bbc8dbdb09d.png)

Reply:
have put it back to what it was and after relaunching the server the date shows fine with that previous method, no idea what happened 🤷

Reply:
config/* are picked up on boot, so if you kept your server running (and you hadn't stopped it for a while) it might not have picked it up. Glad it works now! :star: